builtin/rebase--helper.c

```diff
@@ -12,12 +12,12 @@ static const char * const builtin_rebase_helper_usage[] = {
 int cmd_rebase__helper(int argc, const char **argv, const char *prefix)
 {
 	struct replay_opts opts = REPLAY_OPTS_INIT;
-	unsigned flags = 0, keep_empty = 0, rebase_merges = 0, edit_todo = 0;
```

Comment:
In the commit message you say "`edit-todo`, is added, and the `write-edit-todo` is removed" which is confusing when we see the above line.

It might have been better to use `write_edit_todo` instead of `edit_todo` in the first place.

Reply:
I just rerolled this patch to rename the variable.